src/button.scss

```diff
@@ -409,7 +408,6 @@ $block: #{$fd-namespace}-button;
   &[aria-disabled="true"],
   &.is-disabled {
     @include isDisabled();
-    @include buttonContainer();
```

Comment:
this will introduce another bug. Check this PR and the issue: https://github.com/SAP/fundamental-styles/pull/2597/files